.config/zsh/000.init.zsh##o.Linux,e.zsh

```diff
@@ -0,0 +1,26 @@
+autoload -Uz compinit
+compinit
+
+# ZSH plugins for Linux (installed via git clones, not brew)
```

Comment:
This file here looks fine, but I guess you have missed to rename the existing 000.init.zsh to be Mac related